src/diffbot.js

```diff
@@ -261,10 +265,12 @@ class Diffbot {
           throw new Error('missing name');
         else if (!options.seeds || !options.seeds.length)
           throw new Error('missing seeds');
-
+        let websites = options.seeds.split(' ');
+        const maxToCrawl = websites.length*1000;
+        const maxToProcess = maxToCrawl;
```

Comment:
The Diffbot library is meant to be a general purpose library that behaves exactly as you'd expect it to, given the Diffbot docs. As such, I'd like to keep any parameters, such as maxToCrawl, maxToProcess, maxHops, etc, as they behave by default. This way nobody gets confused when the library adds on extra parameters to their Diffbot call when they did not specify them. If the app needs to specify some parameters, please generate the values for the params you need in the app, and then call the library with them.  

e.g.
```javascript
let crawl = await diffbot.crawl().new({
    name: 'my-diffbot-crawl',
    seeds: websites,
    maxToCrawl: websites.length * 1000,
  });
```